apps/happy-blocks/package.json

```diff
@@ -55,8 +56,5 @@
 		"glob": "^7.1.6",
 		"postcss": "^8.4.5",
 		"webpack": "^5.68.0"
-	},
-	"peerDependencies": {
-		"@wordpress/data": "^8.0.0"
```

Comment:
This shouldn't be a peer, because happy-blocks is an app. As a result, no other npm project will install happy-blocks and provide wp-data for us. (Thankfully, this doesn't cause an issue in practice, because wp/data is a peer request from automattic/components, and because wp/data is provided by WordPress at runtime anyways.)